compiler-plugin/src/main/java/io/ballerina/lib/np/compilerplugin/PromptAsCodeCodeModificationTask.java

```diff
@@ -77,55 +95,95 @@ public class PromptAsCodeCodeModificationTask implements ModifierTask<SourceModi
     private static final Token COLON = createToken(SyntaxKind.COLON_TOKEN);
     private static final Token RIGHT_DOUBLE_ARROW = createToken(SyntaxKind.RIGHT_DOUBLE_ARROW_TOKEN);
     private static final Token COMMA = createToken(SyntaxKind.COMMA_TOKEN);
+    public static final String SCHEMA_ANNOTATION_IDENTIFIER = "schemaAnnot";
+    static final String CALL_LLM = "callLlm";
+    static final String STRING = "string";
+    static final String BYTE = "byte";
+    static final String NUMBER = "number";
 
+    private static Optional<String> npPrefixIfImported = Optional.empty();
```

Comment:
I would leave this as is and set it to `TypeDefinitionModifier` as was done for the other modifier.

Reply:
Without moving it to this level.